alea/submitters/htcondor.py

```diff
@@ -487,6 +503,26 @@ def _add_combine_job(self, combine_i):
 
         return combine_job
 
+    def _add_separate_job(self, combine_i):
+        """Add a separate job to the workflow."""
+        logger.info(f"Adding separate job {combine_i} to the workflow")
+        separate_name = "separate"
+        separate_job = self._initialize_job(
+            name=separate_name,
+            cores=self.request_cpus,
+            memory=self.request_memory * 2,
```

Comment:
What is the motivation for double here?

Reply:
https://github.com/XENONnT/alea/blob/335ff8b153789f201f4afc50685ffe4fbeeba733/alea/submitters/htcondor.py#L476

Reply:
Copied from you.

Reply:
OK I don't think this is a problem. Just curious